src/App/component/CommitsPage.jsx

```diff
@@ -64,108 +65,96 @@ function CommitsPage(prop) {
 
   const [open, setOpen] = useState(false);
   const [isLoading, setLoading] = useState(false);
-  const handleClose = () => {
-    setOpen(false);
-  };
-  const handleToggle = () => {
-    setOpen(!open);
-  };
+  const loadingCommitsEnd = () => {
+    setOpen(false)
+  }
+  const loadingCommitsStart = () => {
+    setOpen(!open)
```

Comment:
```suggestion
    setOpen(true)
```
If the function name has a keyword "Start", suggest setting `open` true directly.